renku/core/management/storage.py

```diff
@@ -124,10 +121,14 @@ def check_external_storage(self):
     def renku_lfs_ignore(self):
         """Gets pathspec for files to not add to LFS."""
         ignore_path = self.path / self.RENKU_LFS_IGNORE_PATH
+        renku_protected_paths = [".renku"]
```

Comment:
This would also match e.g. `data/mydataset/.renku`, which might not be intended (`/.renku` would be to only match the .renku directory). But then again, I can't think of a case where we'd add a file or directory called `.renku` somewhere else and want it to end up in lfs, so it probably doesn't matter.